FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -329,6 +333,12 @@ - (void)activateWithCompletion:(FIRRemoteConfigActivateChangeCompletion)completi
     // New config has been activated at this point
     FIRLogDebug(kFIRLoggerRemoteConfig, @"I-RCN000069", @"Config activated.");
     [strongSelf->_configContent activatePersonalization];
+
+    _rolloutsState = [strongSelf->_configContent activateRolloutMetdata];
+    [[NSNotificationCenter defaultCenter]
```

Comment:
Feel like passing rollouts as [user info](https://developer.apple.com/documentation/foundation/nsnotificationcenter/1410608-postnotificationname) will be eliminate the risk condition of updating `_rolloutsState`. Otherwise we probably need to sync on `_rollouts` to avoid risk condition.